lib/apivore/validator.rb

```diff
@@ -21,8 +22,7 @@ def matches?(swagger_checker)
         send(
           method,
           full_path(swagger_checker),
-          params['_data'] || {},
-          params['_headers'] || {}
+          *RailsShim.request_params(params)
```

Comment:
Looking back, that's a great point.  The `|| {}` check is going to have to move back to `Apivore::Validator` at whatever point Rails 4 is no longer supported, so it makes sense to leave it there.

As for splatting the entire arg set vs. just a subset, I have no strong feelings, so I'll just go with your gut here @ashoda.